src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -80,8 +80,8 @@ private function init($redisClient, $namespace, $defaultLifetime, ?MarshallerInt
      */
     public static function createConnection($dsn, array $options = [])
     {
-        if (0 !== strpos($dsn, 'redis:')) {
-            throw new InvalidArgumentException(sprintf('Invalid Redis DSN: %s does not start with "redis:".', $dsn));
+        if (0 !== strpos($dsn, 'redis:') && 0 !== strpos($dsn, 'rediss:')) {
```

Comment:
you also need to update the regex of the `preg_replace_callback` below

Reply:
OK, @stof , I added tests and modified regexp.